### PR TITLE
feat(split-ui): Slice 3 PR-D — durable loggedRun receipt + log-draft card states (DEC-091)

### PR DESCRIPTION
**Pull request by leehopper**
## Slice 3 PR-D — durable `loggedRun` receipt + log-draft card states

The final PR of Slice 3 (Coach) in the SPLIT / Alpine UI redesign, and the only one that depends on PR-A. Built from the locked, adversarially-verified spec `docs/specs/slice-3-coach/spec.md` § 3 (PR-D) / § 4.3 / DU-6. PR-A (backend `loggedRun`), PR-B (turn kinds), and PR-C (adaptation + safety turns) already shipped.

### What changed
- **`LoggedRunReceipt`** (new) — the durable one-line confirm-ack receipt: `✅ LOGGED — 5.0 km · 25:00 · JUN 29   LOG BOOK →`, `LOG BOOK →` links to `/history`. Rendered by `CoachTextTurn` from the **persisted** `interactive.loggedRun` (DEC-091), so it survives reload — durability comes entirely from the event-sourced turn, never from ephemeral card state.
- **`formatReceiptDate`** (new helper, `transcript-time.helpers.ts`) — guarded: on an unparseable `occurredOn` it omits the date fragment rather than crashing into the Unix epoch (`new Date(null)` → `JAN 1`). `formatDurationSeconds` already existed from PR-B and was reused unchanged.
- **`CoachTextTurn`** — renders the receipt beneath the coach body when `loggedRun` is non-null; `units` defaults to `Kilometers`.
- **`LogConfirmationCard`** — Alpine restyle (`LOG THIS RUN?`, 2×2 grid with STATUS in moss, humanized DATE, **ON-PLAN target distance** added), the **saving** state (dim, buttons locked, `Saving…`, CANCEL hidden), and `log-confirm` / `log-edit` / `log-cancel` testids. Confirm-success keeps the **locked** immediate-dismiss → refetch contract; the optimistic in-card bridge stays deferred (spec § 9 #2).
- **e2e realignment** (`conversation-streaming.spec.ts`) — the existing `/coach` confirm flow now asserts the durable receipt renders + its `LOG BOOK →` href, and reloads to prove reload durability (spec § 7).

### Deliberate spec deviation
The spec § 3 specifies `text-[var(--alp-faint)]` for the card's field labels and ON-PLAN line, but the field labels are **essential text** and the frontend `CLAUDE.md` § Styling is a hard rule: `--alp-faint` "must never carry essential text." `CLAUDE.md` wins — the labels use `text-muted-foreground` (AA-passing). Truly-decorative meta (timestamps, dividers) keeps `--alp-faint` as before.

### Verification
- `npm run build` clean · `npm run test` **937/937** vitest pass · `npm run check-contrast` **44/44** · scoped `eslint` on changed files clean.
- New/extended tests cover every DU-6 bullet + § 6 edge cases: receipt format, LOG BOOK link, `units === Miles`, unparseable-`occurredOn` omission (no epoch), receipt-absent on a historical `loggedRun: null` ack turn, default-units, the card saving state + ON-PLAN target, and confirm → durable receipt.

### Process
Built via an orchestrated sonnet workflow (recon → TDD implement → gate loop → 4-lens adversarial review → challenge → apply). All 7 confirmed review findings were applied; two the automated gate structurally cannot catch were fixed by hand and re-verified: a Playwright API error in the e2e edit (`toHaveTextContent` → `toContainText`) and reverting three unrelated auth/login/register lint-suppression edits that had crept in.

Not built (deferred, per spec): partial-parse `MISSING` card states (backend never emits a partial draft) and the optimistic in-card receipt bridge (§ 9 #2).

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01CnVnYW3bL6bKDKT8sgnh1B

## Summary by CodeRabbit

* **New Features**
  * Added a durable receipt after confirming a workout log, showing distance, duration, date, completion status, and a link to the log book.
  * Added support for displaying workout distances in either kilometers or miles.
  * Improved workout confirmation details, including on-plan targets and clearer saving-state feedback.
* **Bug Fixes**
  * Receipts now persist after page reloads and are safely omitted when workout data is unavailable or dates are invalid.

### CODE REVIEW

File: frontend/src/app/modules/coaching/components/coach-chat.component.spec.tsx [hunk 1]
```diff
@@ -571,6 +571,64 @@ describe('CoachChat', () => {
...
+        proactive: null,
+      },
+    ])
+    streamMock.mockReturnValue(idleStream())
+
+    renderChat()
+
+    // The receipt renders wherever the ack turn sits in the timeline — it is
+    // sourced from the persisted `loggedRun` field, not any in-session card
+    // state, so it survives a fresh GET /timeline (reload) by construction.
```
[the remark this answers is not in the document]
Comment (leehopper) [reply frontend/src/app/modules/coaching/components/coach-chat.component.spec.tsx]: Addressed in 611788c — backticked `GET /timeline`, the mile-conversion expressions, `formatReceiptDate`, and `expectDualThemeParity` across the two spec files. Scoped eslint + vitest stay green.

File: frontend/src/app/modules/coaching/components/coach-text-turn.component.tsx [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+import { LoggedRunReceipt } from './logged-run-receipt.component'
 
 export interface CoachTextTurnProps {
   content: string
   time: string
   /** Appends the clay block-cursor inline at the end of the body when a live stream is still in flight. */
   streaming?: boolean
-  /** Wires into the durable `LoggedRunReceipt` (spec §3 PR-D); undefined when the receipt isn't rendered. */
+  /** Renders the durable `LoggedRunReceipt` beneath the body when non-null; null/undefined render no receipt. */
   loggedRun?: LoggedRunSummaryDto | null
```
[the remark this answers is not in the document]
Comment (leehopper) [reply frontend/src/app/modules/coaching/components/coach-text-turn.component.tsx]: Addressed in cb14b17 — backticked `null` / `undefined` in the JSDoc. eslint + build green.
